### PR TITLE
Added Gumby hello world tutorial

### PR DESCRIPTION
**Pull request by qstokkink**
This PR adds a hello-world tutorial for basic Gumby usage.

### CODE REVIEW

**Comment by qstokkink** [comment]
@devos50 Ready for review!

File: docs/hello_world.rst [hunk 1]
```diff
@@ -0,0 +1,228 @@
+The purpose of this document is to showcase creation of a basic Gumby experiment.
+This document assumes the reader has a basic understanding of creating Dispersy communities.
```
Comment (devos50) [inline docs/hello_world.rst]: Maybe add a reference to the dispersy documentation here? dispersy.readthedocs.org

File: docs/hello_world.rst [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+``gumby/experiments/hello_world/``
+
+This folder will house the aforementioned files (``hello_world.conf``, ``hello_world_module.py`` and ``hello_world.scenario``).
+
+The configuration file
+----------------------
+
+The entry point for your experiment is the configuration (``.conf``) file.
+Configuration files mainly fall into one of two categories:
```
Comment (devos50) [inline docs/hello_world.rst]: "Into one of the two *following* categories"

File: docs/hello_world.rst [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+    ## hello_world.scenario ##
+    # With this we tell Gumby to load the DispersyModule, which takes care of providing a Dispersy instance for us
+    # If you want extended Tribler functionality, you should use gumby.modules.tribler_module.TriblerModule instead
+    &module gumby.modules.dispersy_module.DispersyModule
+    
+    # This tells Gumby to load our hello_world_module.py file's HelloWorldModule class
+    &module experiments.hello_world.hello_world_module.HelloWorldModule
+    
+    # At 0 seconds, make sure our HelloWorldCommunity does not communicate with the outside world
```
Comment (devos50) [inline docs/hello_world.rst]: I would say: At 0 seconds into the experiment ... (also for the subsequent comments)

File: docs/hello_world.rst [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+    @1:15 stop
+
+If you find yourself writing the same statements over and over, you can use ``&include some_other.scenario`` to include the entirety of a different scenario file.
+As a final note: the timestamps can go up to hours (``hours:minutes:seconds``), though currently most experiments in Gumby only use seconds.
+
+The module file
+---------------
+
+The module file is what provides the functionality for the events generated by the scenario file.
+It is common to use the ``_module`` postfix when naming your module python file.
```
Comment (devos50) [inline docs/hello_world.rst:168]: I would add: "An example of a Python module file is given below".

File: docs/hello_world.rst [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+    from gumby.modules.community_experiment_module import CommunityExperimentModule
+    from gumby.modules.community_launcher import CommunityLauncher
+    from gumby.modules.experiment_module import static_module
+    from gumby.modules.isolated_community_loader import IsolatedCommunityLoader
+
+    from Tribler.dispersy.community import Community
+    from Tribler.dispersy.conversion import DefaultConversion
+
+
+    class HelloWorldCommunityLoader(IsolatedCommunityLoader):
```
Comment (devos50) [inline docs/hello_world.rst:183]: Please reference the document explaining the loader/launcher.

File: docs/hello_world.rst [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+        def hello(self):
+            print "Hello human!"
+
+        @experiment_callback
+        def extended_hello(self, repetitions, separator=" "):
+            print separator.join(["Hello human!"]*int(repetitions))
+
+Ordinarily one would have his ``@experiment_callback`` actually do something with the loaded community (``self.community``).
+For the sake of keeping this example short, these callbacks only perform print statements.
+Furthermore, why one isolates Dispersy Communities and how the Communities are made will also remain outside of the scope of this document.
```
Comment (devos50) [inline docs/hello_world.rst]: Please don't capitalize `Communities`

File: docs/hello_world.rst [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+
+You are now ready to run your experiment! You can do so by making the following call (make sure you followed the README setup instructions correctly):
```
Comment (devos50) [inline docs/hello_world.rst]: "By running the following command"

**Review by devos50 — CHANGES_REQUESTED** [review]

**Comment by qstokkink** [comment]
@devos50 Fixed 👍

**Review by devos50 — APPROVED** [review]

**Pull request merged**